session/object.go

```diff
@@ -1,166 +1,379 @@
 package session
 
 import (
+	"bytes"
+	"crypto/ecdsa"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/nspcc-dev/neofs-api-go/v2/refs"
 	"github.com/nspcc-dev/neofs-api-go/v2/session"
+	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
+	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
 	"github.com/nspcc-dev/neofs-sdk-go/object/address"
+	"github.com/nspcc-dev/neofs-sdk-go/user"
 )
 
-// ObjectContext represents NeoFS API v2-compatible
-// context of the object session.
+// Object represents token of the NeoFS Object session. A session is opened
```

Comment:
`Container` and `Object` descriptions are the same, don't we want to add the difference between them into the comment

Reply:
I thought about that, but decided to not do like this since different sessions are mostly used in unrelated contexts. I think we can later create an abstract type `Token`, and inherit it in `Object` and `Container` types. I can try to do it here or leave an issue. What u think?

Reply:
i do not mind a separate issue or leaving it as it is. just thought that such description may confuse